docs/README.md

```diff
@@ -97,6 +96,7 @@ For now `ContactType` supports following enums:
 - phone
 - twitter
 - web
+- codepen
```

Comment:
sigh, it _was_ alphabetical...

Reply:
Oops, sorry, I didn't notice😓

[style: alphabetize contact types](https://github.com/vuepressjs/vuepress-theme-blog/commit/863e5fb16298d417c7d1e0e3e053f09678267bdf)